l1-contracts/contracts/bridge/ntv/NativeTokenVault.sol

```diff
@@ -186,7 +186,7 @@ abstract contract NativeTokenVault is INativeTokenVault, IAssetHandler, Ownable2
     function _bridgeBurnBridgedToken(
         uint256 _chainId,
         bytes32 _assetId,
-        address _prevMsgSender,
+        address _account,
```

Comment:
I don't like account. Is it the sender, the receiver, etc?

Reply:
`_prevMsgSender` was just not correct so I make it generic but correct, let me think about better naming

Reply:
@vladbochok why was it not correct? the same `_prevMessageSender` is used everywhere include asset router

Reply:
`_prevMsgSender` is actually more generic as it literally describes what this is

Reply:
What is the previous `msg.sender`?

Reply:
The problem is that it is used in this context:
```
functionA() {
previousMsgSender = msg.sender
functionB(previousMsgSender)
}

functionB(previousMsgSender) {
functionC(previousMsgSender)
}
...
```

Reply:
`original_caller`?